models/todo.js

```diff
@@ -0,0 +1,14 @@
+module.exports = (sequelize, DataTypes) => {
+    var Sequelize = require('sequelize');
+    var sequelize = new Sequelize(process.env.DATABASE_URL)
+    
+    return sequelize.define('Todo', {
+        title: {
+            type: Sequelize.STRING,
+            unique: false
```

Comment:
good catch. I think this was an unnecessary thing I left in after pulling code from the Sequelize docs. Removed it and the tests work :+1: